docs/antrea-network-policy.md

```diff
@@ -689,6 +690,61 @@ The following kubectl commands can be used to retrieve CG resources:
     kubectl get cg.core.antrea.tanzu.vmware.com
 ```
 
+## Select Namespace by Name
+
+Kubernetes NetworkPolicies and Antrea-native NetworkPolicies allow selecting
```

Comment:
nit: s/Antrea-native NetworkPolicies/Antrea-native policies
for consistency

Reply:
done